src/components/Books.js

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable import/extensions */
 import React from 'react';
-import MyBook from './BookItem';
-import Form from './BookForm';
+import MyBook from './bookItem.js';
+import Form from './bookForm.js';
 
```

Comment:
You have done very good work on this project but I realized your project couldn't build because some modules could not be resolved. With some tweaking I realized that in this snippets, your import statements should be something like this: 

```sh
   import MyBook from './BookItem.js';
   import Form from './BookForm.js';
```

Your project should build and run properly if you correct this 👍